### PR TITLE
Add Govee BLE integration

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Add Govee BLE integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/75631
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3548
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/govee_ble.markdown [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Govee BLE
+description: Instructions on how to integrate Govee BLE devices into Home Assistant.
+ha_category:
+  - Sensor
+ha_zeroconf: true
```
Comment (bdraco) [inline source/_integrations/govee_ble.markdown]:
```suggestion
ha_bluetooth: true
```

File: source/_integrations/govee_ble.markdown [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ha_domain: govee_ble
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Integrates [Govee](https://www.govee.com/) BLE devices into Home Assistant.
+
+## Supported devices
```
Comment (MartinHjelmare) [inline source/_integrations/govee_ble.markdown:24]: I'd put this in the order:

- Configuration
- Paragraph about setting up bluetooth to discover devices.
- Supported devices.

Comment (bdraco) [reply source/_integrations/govee_ble.markdown:24]: a4be232a18bcfe3dcaa34ccf154369ab0f0828ea

**Review by MartinHjelmare — APPROVED** [review]

**Comment by bdraco** [comment]
Thanks

**Pull request merged**